python/cudf/setup.py

```diff
@@ -61,8 +62,13 @@
         library_dirs=(
             pa.get_library_dirs()
             + [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
+            + [
+                os.path.join(
+                    os.path.join(os.sys.prefix, "libcudf_kafka"), "lib"
+                )
+            ]
         ),
-        libraries=["cudf"] + pa.get_libraries(),
+        libraries=["cudf", "cudf_kafka"] + pa.get_libraries(),
```

Comment:
We should build a separate `cudf_kafka` Python library instead of adding it into the `cudf` Python library.